src/keyValueStore.js

```diff
@@ -52,7 +52,8 @@ class KeyValueStore {
   async _sync (numRemoteEntries) {
     this._requireLoad()
     // let toid = null
-    if (numRemoteEntries <= this._db._oplog.values.length) return Promise.resolve()
+    const numEntriesDefined = !(numRemoteEntries === null || numRemoteEntries === undefined)
+    if (numEntriesDefined && numRemoteEntries <= this._db._oplog.values.length) return Promise.resolve()
```

Comment:
Why is `numEntriesDefined` needed? couldn't the if statement simply be this?
```js
if (numRemoteEntries && numRemoteEntries <= this._db._oplog.values.length) return Promise.resolve()
```

Reply:
problems is 0 vs undefined vs null in js - some examples 
null < 1  = true
undefined < 1 = false 
1 && 1 < 2 = true 
0 && 0 < 2 = 0  (which is falsey)

yeah caused some trouble for a little while, so just made it explicit

Reply:
ah, I see! Annoying :P